src/material-examples/typings.d.ts

```diff
@@ -0,0 +1 @@
+declare var module: {id: string};
```

Comment:
Alright that makes sense then. For the examples you removed the `module.id`. You can remove the typings file therefore too